how-to/create-a-snapshot-source-client/OpenFin.Web.Platform/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "profiles": {
+    "OpenFin.Web.Platform": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "dotnetRunMessages": true,
+      "applicationUrl": "http://localhost:8080"
```

Comment:
Should we use a different port so each starter can run at the same time (we've done this with container and others)?

Reply:
Updated all 8080s to 4040